src/webgpu/util/color_space_conversion.ts

```diff
@@ -0,0 +1,190 @@
+import { assert } from '../../common/util/util.js';
+
+/**
+ * The color space conversion function definations are fully aign with
+ * CSS Color Module Level 4 Sample Code: https://drafts.csswg.org/css-color/#color-conversion-code
+ */
+
+/**
+ * Simple matrix (and vector) multiplication
+ * Warning: No error handling for incompatible dimensions!
+ * @author Lea Verou 2020 MIT License
+ */
+// A is m x n. B is n x p. product is m x p.
+function multiplyMatrices(A: Array<Array<number>>, B: Array<Array<number>>) {
```

Comment:
multiplyMatrices is probably generally more useful. We can put it somewhere else like utils/math.ts ?

Please also add all exported helpers to helper_index.txt